src/utils/appinsights.ts

```diff
@@ -95,3 +95,22 @@ export function sessionIdPreprocessor(
   }
   return true;
 }
+
+export enum StartupEventName {
+  SERVER = "api-backend.httpserver.startup",
+  SPID = "api-backend.spid.config"
+}
+
+export const trackStartupTime = (
+  telemetryClient: appInsights.TelemetryClient,
+  type: StartupEventName,
+  timeMs: bigint
+): void => {
+  telemetryClient.trackEvent({
+    name: type,
+    properties: {
+      time: timeMs.toString()
+    },
+    tagOverrides: { samplingEnabled: "false" }
+  });
```

Comment:
@gunzip @balanza we can use `trackMetric` instead `trackEvent`, maybe is better for this kind of misurement
```suggestion
  telemetryClient.trackMetric({
    name: type,
    tagOverrides: { samplingEnabled: "false" },
    value: parseInt(timeMs.toString(), 10)
  });
```